etl/steps/data/meadow/papers/2023-01-04/nemet_2009.py

```diff
@@ -0,0 +1,53 @@
+"""Load snapshot of Nemet (2009) data and create a table.
+
+"""
+
+import pandas as pd
+from owid import catalog
+
+from etl.helpers import PathFinder
+from etl.steps.data.converters import convert_snapshot_metadata
+
+__file__ = "/Users/prosado/Documents/owid/repos/etl/etl/steps/data/meadow/papers/2023-01-04/nemet_2009"
+
+
+# Get paths and naming conventions for current data step.
+paths = PathFinder(__file__)
```

Comment:
Related to @Marigold comment in this file, I ran `etl papers` and I have an error in this line. I assume because of the directory in `__file__`:
```
  File "/home/parriagadap/etl/etl/steps/data/meadow/papers/2023-01-04/nemet_2009.py", line 15, in <module>
    paths = PathFinder(__file__)
  File "/home/parriagadap/etl/etl/helpers.py", line 102, in __init__
    raise CurrentFileMustBeAStep
etl.helpers.CurrentFileMustBeAStep: Current file must be an ETL step.
```

Reply:
Yes, I hardcoded `__file__` just to quickly test something on a notebook, it should not be there. I've removed it. Thanks.